packages/sdk/Dockerfile

```diff
@@ -180,5 +184,7 @@ ADD --chmod=644 \
 RUN tar -xJf /usr/share/cartesi-machine/images/linux-headers.tar.xz -C / && \
     rm /usr/share/cartesi-machine/images/linux-headers.tar.xz
 
+COPY --from=espresso-dev-node /usr/bin/espresso-dev-node /usr/bin/
```

Comment:
any reason to be at `/usr/bin` and not `/usr/local/bin` as the other binaries?

Reply:
No, will fix that.